src/lib.rs

```diff
@@ -244,10 +246,48 @@ impl OpenatDirExt for openat::Dir {
         destname: P,
         mode: libc::mode_t,
     ) -> io::Result<FileWriter> {
-        let tmpf = self.new_unnamed_file(mode)?;
+        let (tmpf, name) = if let Some(tmpf) = self.new_unnamed_file(mode).ok() {
+            (tmpf, None)
+        } else {
+            // FIXME allow this to be configurable
+            let (tmpf, name) = tempfile_in(self, ".tmp", ".tmp", mode)?;
```

Comment:
Using `.tmp` for both the prefix and suffix seems a bit overdescriptive, but of course fine as is too.

Reply:
Yeah, just trying hard to ensure the file is ignored by default.